packages/patternfly-4/react-core/src/components/Avatar/Avatar.tsx

```diff
@@ -1,7 +1,9 @@
-import React from 'react';
-import styles from '@patternfly/patternfly-next/components/Avatar/avatar.css';
+import * as React from 'react';
+import * as styles from '@patternfly/patternfly-next/components/Avatar/avatar.css';
 import { css } from '@patternfly/react-styles';
-import PropTypes from 'prop-types';
+import * as PropTypes from 'prop-types';
+
+let _styles: any = styles;
```

Comment:
should be able to place a d.ts file somewhere under src with the following
```ts
declare module '*.css' {
  const styles: any;
  export default styles;
}
```

Reply:
Done add a typings folder at the same level as src in react-core